### PR TITLE
Implement exchange paging

### PR DESCRIPTION
**Pull request by Pliner**
The same as #148, but for exchanges

### CODE REVIEW

File: Source/EasyNetQ.Management.Client/IManagementClient.cs [hunk 1]
```diff
@@ -82,6 +82,31 @@ public interface IManagementClient : IDisposable
     /// <returns></returns>
     Task<IReadOnlyList<Exchange>> GetExchangesAsync(CancellationToken cancellationToken = default);
 
+    /// <summary>
+    ///     A list of all exchanges.
```
Comment (sungam3r) [inline Source/EasyNetQ.Management.Client/IManagementClient.cs:86]: Copy-pasted from GetExchangesAsync

File: Source/EasyNetQ.Management.Client/IManagementClient.cs [hunk 2]
```diff
@@ -82,6 +82,31 @@ public interface IManagementClient : IDisposable
...
+    /// </summary>
+    /// <param name="vhostName"></param>
+    /// <param name="cancellationToken"></param>
+    /// <returns></returns>
+    Task<IReadOnlyList<Exchange>> GetExchangesAsync(string vhostName, CancellationToken cancellationToken = default);
+
+    /// <summary>
+    ///     A list of all exchanges.
```
Comment (sungam3r) [inline Source/EasyNetQ.Management.Client/IManagementClient.cs:102]: the same here

Comment (Pliner) [reply Source/EasyNetQ.Management.Client/IManagementClient.cs:102]: https://github.com/EasyNetQ/EasyNetQ.Management.Client/commit/fd3df85311400dd1e4b457a202943bedf92315bf

File: Source/EasyNetQ.Management.Client/IManagementClient.cs (hunk 1 above)
Comment (Pliner) [reply Source/EasyNetQ.Management.Client/IManagementClient.cs:86]: https://github.com/EasyNetQ/EasyNetQ.Management.Client/commit/fd3df85311400dd1e4b457a202943bedf92315bf

**Pull request merged**